java/compaction/compaction-job-execution/src/main/java/sleeper/compaction/jobexecution/CompactSortedFiles.java

```diff
@@ -56,20 +55,15 @@
 import java.time.LocalDateTime;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Map;
 import java.util.NoSuchElementException;
 import java.util.Objects;
-import java.util.function.Function;
-import java.util.stream.Collectors;
 
 import static sleeper.core.metrics.MetricsLogger.METRICS_LOGGER;
 import static sleeper.sketches.s3.SketchesSerDeToS3.sketchesPathForDataFile;
 
 /**
  * Executes a compaction {@link CompactionJob}, i.e. compacts N input files into a single
- * output file (in the case of a compaction job within a partition) or into
- * two output files (in the case of a splitting compaction job which reads
- * files in one partition and outputs files to the split partition).
+ * output file (in the case of a compaction job within a partition).
```

Comment:
Delete "(in the case of a compaction job within a partition)".

Reply:
Done, thanks